Sources/EdgeNetworkHandlers/RequestBuilder.swift

```diff
@@ -23,8 +23,8 @@ class RequestBuilder {
     /// Control character used at the end of each response fragment. Response streaming is enabled when both `recordSeparator` and `lineFeed` are non nil.
     private var lineFeed: String?
 
-    /// The Experience Cloud ID to be sent with this request
-    var experienceCloudId: String?
+    /// XDM payloads to be attached to the request
+    var xdmPayloads: [[String: AnyCodable]] = []
```

Comment:
I made the change as suggested locally, but I actually find the original name of `xdmPayloads` to be more concise and self-explanatory.

Example:
`requestBuilder.xdmPayloads = [AnyCodable.from(dictionary: identityState) ?? [:]]`
vs
`requestBuilder.requestXdmPayload = [AnyCodable.from(dictionary: identityState) ?? [:]]`

1. In many cases `request` in the variable name for the XDM payloads is redundant given it's attached to the request builder.
2. The original name indicates its plural, indicating it could contain multiple XDM payloads.